.travis.yml

```diff
@@ -1,11 +1,7 @@
+sudo: false
 language: python
-
 python:
-  - 3.6
-
-install:
-  - python -m pip install flake8 pydocstyle
-
-script:
-  - flake8 src
-  - pydocstyle src
+- "2.7"
+- "3.6"
+install: pip install tox-travis
+script: tox
```

Comment:
this nice tox-travis plugin makes travis soooo simple.  We only are running against 2.7 and 3.6, but can add other versions easily that are in `tox.ini`